### PR TITLE
Fix bug when body is empty or null

### PR DESCRIPTION
**Pull request by MikeWilliams-UK**
Fixes for issue "Errors when body is empty #310"

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-PullRequest0312.89.build.528 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/22921994) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/b360d97f33 by @)

File: src/ServiceBusExplorer.sln.DotSettings [hunk 1]
```diff
@@ -590,6 +590,9 @@ II.2.12 &lt;HandlesEvent /&gt;&#xD;
 	<s:String x:Key="/Default/CodeStyle/Naming/XamlNaming/UserRules/=XAML_005FFIELD/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AaBb" /&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/XamlNaming/UserRules/=XAML_005FRESOURCE/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AaBb" /&gt;</s:String>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpFileLayoutPatternsUpgrade/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpKeepExistingMigration/@EntryIndexedValue">True</s:Boolean>
```
Comment (SeanFeldman) [inline src/ServiceBusExplorer.sln.DotSettings]: Please revert this change.

Comment (MikeWilliams-UK) [reply src/ServiceBusExplorer.sln.DotSettings]: Ok I will do that on Monday when I am back at work.

**Comment by MikeWilliams-UK** [comment]
Not sure why R# decided to change the file. My understanding of these files are that R# should apply them as solution specific settings. As I didn't knowingly modify any settings the file should not have changed. My bad also for not spotting this and excluding it form the check-in

**Comment by SeanFeldman** [comment]
No worries. The file can get modified locally, just need to make sure not
to stage/commit those changes.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-PullRequest0312.90.build.529 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/22960713) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/38478e1ede by @MikeWilliams-UK)

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-PullRequest0312.91.build.530 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/22960770) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/44c390984c by @MikeWilliams-UK)

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-PullRequest0312.92.build.531 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/22960824) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/3c6c090532 by @MikeWilliams-UK)

**Comment by paolosalvatori** [comment]
Thanks @MikeWilliams-UK!
